locales/es/front/credits.json

```diff
@@ -0,0 +1,95 @@
+{
+  "button": {
+    "AddCredit": "Agregar\nCrédito"
+  },
+  "error": {
+    "SomePlanChangedWasNotSaved": "Algunos cambios no se guardaron debido a errores."
+  },
+  "form": {
+    "hint": {
+      "Duration": "Meses",
+      "FeeFromSyncAccountsAll": "El pago automático no funcionará con cuentas sincronizadas.",
+      "FeeFromSyncAccountsSome": "El pago automático no funcionará con cuentas sincronizadas."
+    },
+    "label": {
+      "Duration": "Duración",
+      "EndDate": "Fecha de finalización",
+      "FeeFromAccounts": "Pagar automáticamente desde cuentas",
+      "Name": "Nombre",
+      "OnlyCurrency": "No agregar a la cuenta",
+      "PlanType": "Tipo de plan",
+      "Rate": "Tasa",
+      "ReceiveCreditToAccount": "Recibir crédito a la cuenta",
+      "RemainToFee": "Permanecer en la tarifa",
+      "StartDate": "Fecha de inicio",
+      "TotalCreditAmount": "Monto total del crédito",
+      "TotalPayAmount": "Monto total del pago"
+    },
+    "planType": {
+      "Ann": "Anualidad",
+      "Diff": "Diferencial"
+    },
+    "title": {
+      "CreateCredit": "Añadir crédito",
+      "EditCredit": "Editar crédito"
+    }
+  },
+  "formOperation": {
+    "label": {
+      "DealWithAccounts": "Afecta las cuentas",
+      "LinkToPlan": "Enlace al plan",
+      "LinkToPlanItem": "{date}: {paid} {ofTotal} ({percent}%)",
+      "NoLinkToPlan": "fuera del plan",
+      "SelectTransaction": "Seleccione transacción...",
+      "TransactionLinked": "Transacción vinculada por {amount} en {date}{ifAccountValid, seleccione, 0{ with another account} otra{}}",
```

Comment:
Please, check ICU `{select}` syntax: https://formatjs.io/docs/core-concepts/icu-syntax/#select-format
There are several `{select}`s is all the files.

Reply:
Hello, I made the corrections.